frontend/packages/core/src/stories/buttongroup.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
super nit: can we add a hyphen to the name just for readability? e.g `button-group`